daemon/logic/worklog.go

```diff
@@ -100,13 +107,30 @@ func (w *Worklog) Resolve(ctx context.Context, n *observer.Notifier,
 		return nil, nil
 	}
 
-	return w.handlers[item.Type()].resolve(ctx, n, orgID, item)
+	handler := w.handlers[item.Type()]
+	result, err := handler.resolve(ctx, n, orgID, item)
+
+	if err != nil {
+		result = &apitypes.WorklogResult{
```

Comment:
ya can do. it's just to differentiate between a fatal error in the daemon/registry vs an error in an individual worklog item; so if you're resolving many at once, you can keep going.